pkgs/by-name/in/inori/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "inori";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "eshrh";
+    repo = "inori";
+    tag = "v${version}";
+    hash = "sha256-D3umpem3Vj/T/UECe0qW7+qUpPgIJcRphIniURptgGE=";
+  };
+
+  cargoHash = "sha256-37rqKr+KtSuTI7pyah3nDWYnuusDyBdXjBHSrficPcE=";
+
+  meta = {
+    description = "Mpd client";
+    homepage = "https://github.com/eshrh/inori";
+    changelog = "https://github.com/eshrh/inori/releases/tag/${src.rev}";
```

Comment:
```suggestion
    changelog = "https://github.com/eshrh/inori/releases/tag/v${version}";
```

Reply:
Could also use `src.tag` here to avoid prefixing the `v` again